tests/conftest.py

```diff
@@ -17,6 +17,27 @@
 def pytest_runtest_setup():
     logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
 
+
+@fixture(scope='session')
```

Comment:
Ah, this is slick using the engine fixture at session scope and the session fixture at module scope. Good, good.